src/iiif/tools.py

```diff
@@ -170,18 +174,26 @@ def get_info_from_iiif_url(iiif_url, source_file):
         raise InvalidIIIFUrlError(f"Invalid iiif url: {iiif_url}")
 
 
-def img_is_public(metadata, document_barcode):
+def img_is_public_copyright(metadata, document_barcode):
+    """
+    Return if document is public and has copyright.
+    If it is not public the copyright is not used en returned as unknown
+    """
+    public = None
+    copyright1 = None
     if metadata['access'] != settings.ACCESS_PUBLIC:
-        return False
-
+        public = False
     for meta_document in metadata['documenten']:
         if meta_document['barcode'] == document_barcode:
             if meta_document['access'] == settings.ACCESS_PUBLIC:
-                return True
+                public = True
```

Comment:
Als een dossier niet publiek is maar een document wel, dan geeft deze functie nu een True voor public, waar dat eigenlijk False zou moeten zijn. Gezien deze functie nu complexer is geworden lijkt me een unit test hiervoor een goed idee.